### DISCUSSION TITLE
is vmess without TLS safe?

### DISCUSSION THREAD

**Discussion by fr3shn3ss** [v2fly/v2ray-core#2131] [Q&A]
Hello all, 

Is using vmess without TLS safe? Is it unencrypted? 

![image](https://user-images.githubusercontent.com/32623761/200174673-2bf2b274-29ba-4908-942e-59a7e83f8157.png)

**Comment by yicixin** [comment]
It's not safe, since my vps base on tls+ws+web was baned port 443 recently,  i changed the v2ray to a new port without tls,  just 2 or 3 days, the new port was baned too.

**Reply by fr3shn3ss** [reply]
I want to know if V2ray encrypts the incoming data to the server or not if there is no TLS. I don't have any problem with getting the IP or port banned

As an example, can my ISP listen to my whatsapp calls or see what I send?

**Reply by Jinnrry** [reply]
If you use vmess protocol, all data package will be encrypt. What's more your ISP can't know what you send. But maybe , ISP will know you are send vmess data by v2ray.

**Reply by fr3shn3ss** [reply]
If it's already encrypted, what's the point of adding TLS to vmess?

**Reply by Jinnrry** [reply]
Because most of the data in the Internet is TLS encrypted, using TLS encryption can prevent the network from being blocked by GFW.

If you not in China, you don't need TLS.

**Reply by Jinnrry** [reply]
https://www.v2fly.org/en_US/developer/protocols/vmess.html#%E5%87%BD%E6%95%B0

You can learn more infomation about Vemss in here.
